App.tsx

```diff
@@ -36,15 +39,26 @@ function CustomDrawerContent({ navigation }: any) {
                 onPress={() => navigation.navigate("Form")}
                 labelStyle={styles.drawerLabel}
             />
+            <OpenAppCounter />
         </DrawerContentScrollView>
     );
 }
 
 //---------------------------------
 export default function App() {
-    const [isOpen, setOpen] = useState(false);
-    const isLoadingComplete = useCachedResources();
-    const ThemeContext = React.createContext("peru");
+    const incrementAppCounter = async () => {
+        try {
+            const value = await AsyncStorage.getItem(STORAGE_KEY);
+            if (value !== null) {
+                const new_val = parseInt(value) + 1 + "";
```

Comment:
`num + ""` to taki trochę brzydki sposób na robienie stringa. 
Staraj się raczej używać albo `.toString()` albo `${num}`:
```ts
await AsyncStorage.setItem(STORAGE_KEY, new_val.toString());
```
lub
```ts
const new_val = `${parseInt(value) + 1}`;